xproj/index.py

```diff
@@ -64,7 +64,13 @@ def from_variables(
             raise ValueError("can only create a CRSIndex from one scalar variable")
 
         # TODO: how to deal with different CRS in var attribute vs. build option?
-        crs = var.attrs.get("spatial_ref", options["crs"])
+        crs = var.attrs.get("spatial_ref", var.attrs.get("crs_wkt", options.get("crs", None)))
+
+        if crs is None:
+            raise ValueError(
+                "spatial_ref attribute not dected detected on provided variable {varname!r}"
```

Comment:
```suggestion
                "spatial_ref attribute not detected on provided variable {varname!r}"
```

Also `varname` has not been defined.

Reply:
Could you add a `pytest.raises` test for this error, please?